app/services/publishers/vacancies/vacancy_step_process.rb

```diff
@@ -20,8 +20,18 @@ def initialize(current_step, vacancy:, organisation:, session:)
     })
   end
 
-  def validatable_steps
-    steps - %i[documents review]
+  def validatable_steps(top_level: false)
```

Comment:
I'm skeptical of including a parameter that is by default false and is not set to true anywhere but the tests. What is its purpose?

Reply:
I don't think I originally intended it to be that way.  The step grouping is a bit of a bodge from the look of it, so doing it from outside the class would require reaching inside a bit to know what to remove from the set, but I agree it probably shouldn't be there if it's not used elsewhere, though I do have a preference for leaving useful functionality in place, even if it's not used.

Reply:
Reasonable! Leaving conversation up in case others want to take sides.

Reply:
I've removed this.